core/client/gobindClient/multiLog.go

```diff
@@ -0,0 +1,65 @@
+package gobindClient
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/google/keytransparency/core/client/kt"
+)
+
+var multiLogWriter = MultiIoWriter{[]io.Writer{os.Stderr}}
+
+func init() {
+	kt.Vlog = log.New(&multiLogWriter, "", log.LstdFlags)
+}
+
+// Local copy of io.Writer interface which can be implemented in Java. Used to redirect logs.
+type LogWriter interface {
+	Write(p []byte) (n int, err error)
+}
+
+type MultiIoWriter struct {
+	writers []io.Writer
+}
```

Comment:
This belongs in it's own package. `client/mutlilogger` 
Then you can do `multilogger.New`